x-pack/plugins/lens/public/indexpattern_datasource/loader.ts

```diff
@@ -85,6 +85,7 @@ export function convertDataViewIntoLensIndexPattern(dataView: DataView): IndexPa
   return {
     id: dataView.id!, // id exists for sure because we got index patterns by id
     title,
+    name: name ? name : title,
```

Comment:
nit: name ?? title

Reply:
changed

Reply:
This breaks if I use `name ?? title` as `??` does not do a empty string check and we need a falsy check here